src/components/aktivitetskrav/vurdering/BegrunnelseTextarea.tsx

```diff
@@ -0,0 +1,22 @@
+import { Textarea, TextareaProps } from "@navikt/ds-react";
+import { FlexColumn } from "@/components/Layout";
+import React, { ForwardedRef } from "react";
+
+export const begrunnelseMaxLength = 1000;
+
+const BegrunnelseTextarea = (
```

Comment:
Måtte bruke `forwardRef` her for at det skulle funke å bruke denne komponenten fra parenten med react-hook-form sin `{...register()}`. Kunne evt droppet denne komponenten og heller "duplisert" dette inn i hvert enkelt skjema. :thinking:

Reply:
Jeg har ikke hørt om dette før, så litt vanskelig å si om det er bedre  duplisere koden eller ikke 🤷 
Det er jo kanskje dumt med kode jeg ikke helt skjønner, men det ser ut til å være i tråd med docen da 🤔 
https://react.dev/reference/react/forwardRef

Reply:
Ja, det er vel egentlig ganske standard React-greie, bare at vi ikke bruker det så mye :man_shrugging: